raster/r.slope.aspect/r.slope.aspect.html

```diff
@@ -279,7 +279,7 @@ <h2>REFERENCES</h2>
 <li> Hofierka, J., Mitasova, H., Neteler, M., 2009. <i>Geomorphometry in GRASS GIS.</i>
 In: Hengl, T. and Reuter, H.I. (Eds), <i>Geomorphometry: Concepts, Software, Applications. </i>
 Developments in Soil Science, vol. 33, Elsevier, 387-410 pp,
-<a href="http://www.geomorphometry.org">http://www.geomorphometry.org</a></li>
+<a href="https://www.geomorphometry.org">https://www.geomorphometry.org</a></li>
```

Comment:
DOI of article is https://doi.org/10.1016/S0166-2481(08)00017-2, I found the full volume 33 https://www.sciencedirect.com/bookseries/developments-in-soil-science/vol/33/suppl/C

Reply:
DOI added.